### PR TITLE
Lowercase emails + more logging

### PR DESCRIPTION
**Pull request by alexandrelaplante**

### CODE REVIEW

File: django_saml2_auth/views.py [hunk 1]
```diff
@@ -167,6 +167,7 @@ def _create_new_user(username, email, firstname, lastname):
 def acs(r, metadata_id):
     metadata = SamlMetaData.objects.filter(pk=metadata_id).first()
     if not metadata:
+        print("Denied because missing metadata")
```
Comment (ricardo-passthrough) [inline django_saml2_auth/views.py]: can we use loggers here instead?

Comment (alexandrelaplante) [reply django_saml2_auth/views.py]: the library doesn't have those as a dependency

Comment (ricardo-passthrough) [reply django_saml2_auth/views.py]: I mean, `logging.getLogger(__name__)` from python

Comment (alexandrelaplante) [reply django_saml2_auth/views.py]: nevermind TIL logger is part of the basic library

**Review by jpmanthorp — APPROVED** [review]
LGTM

**Pull request merged**